spec/actions/bump_version_update_changelog_create_pr_action_spec.rb

```diff
@@ -202,6 +202,421 @@
       )
     end
 
+    it 'fails trying to append PHC version if is_prerelease is true' do
+      # Arrange
+      new_version_provided = new_version
+      is_prerelease = true
+
+      # Act, Assert
```

Comment:
These tests are nice but I'm wondering if it's worth delimiting them with these comments... Not a strong opinion though, I'm ok leaving as is.

Reply:
No strong opinion either, removed! https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/67/commits/07db6a548a71e9f1e86bacdf24933dfa28ab5a95